src/data/faq_de.json

```
@@ -417,6 +417,15 @@
                             "Der digitale Impfnachweis wird in der Arztpraxis oder in einem Impfzentrum generiert. Nach Eingabe oder Übernahme der Daten wird Ihnen ein Impfbescheinigungstoken erstellt (QR-Code), den Sie direkt mit der Corona-Warn-App (CWA) einscannen können oder auf einem Papierausdruck mitbekommen und später mit der CWA einscannen können. Sie können dann Ihre digitalen Impfnachweise auf Ihrem Smartphone verwalten."
                         ]
                     },
+                    {
+                        "title": "Was ist eine Auffrischimpfung und wie stellt sie sich in der CWA dar?",

```

Change title (sounds more "normal"). 

```suggestion
                        "title": "Was ist eine Auffrischimpfung und wie wird sie in der CWA dargestellt?",
```

---

@Ein-Tim ok, sounds better, thanks again!